content/en/infrastructure/resource_catalog/policies/_index.md

```diff
@@ -57,11 +56,11 @@ Click the new policy to review all non-compliant resources and filter them by re
 Custom policies let you define a target resource attribute and a desired value, providing flexibility in creating policies for your cloud resources without requiring complex query languages. The following features are available:
 
 * **Access data in nested attributes**:
-Validate more of your configurations (for example mandate that `TLS 1.2` stored in a nested property is used for Amazon CloudFront). 
+Validate more of your configurations (for example, require that `TLS 1.2`, which is data stored in a multi-level property, is used for Amazon CloudFront). 
 * **Use advanced condition matching**:
-Use operators like `\>, \<, \!=` (for example enforcing Kubernetes version \> 1.25).  
+Use advanced condition matching: Use operators like `>, <`, or `!=` (for example, enforcing Kubernetes version > 1.25).  
 * **Use multi-attribute logic**:
-Chain multiple attributes in one policy (for example requiring AWS CloudTrail logging *and* multi-region to be enabled).
+Use multi-attribute logic: Chain multiple attributes in one policy (for example, require AWS CloudTrail logging _and_ multi-region to be enabled).
```

Comment:
```suggestion
Chain multiple attributes in one policy (for example, require AWS CloudTrail logging _and_ multi-region to be enabled).
```

Reply:
This was a suggestion from the PM so will leave this one as is but thank you for the suggestion!